roles/common/tasks/main.yaml

```diff
@@ -62,6 +62,16 @@
         - tfstate.gke_pod_subnet_cidr is defined
         - tfstate.gke_pod_subnet_cidr.value|length > 0
         - (tfstate.gke_pod_subnet_cidr.value) not in LOADBALANCER_SOURCE_RANGES
+    - name: tfstate - Add AKS pod cidr to LOADBALANCER_SOURCE_RANGES # noqa: name[casing]
+      set_fact:
+        LOADBALANCER_SOURCE_RANGES: "{{ LOADBALANCER_SOURCE_RANGES + [tfstate.aks_pod_cidr.value] }}"
```

Comment:
Now we have a section for GKE and AWS. This again should be `cluster_pod_subnet_cidr` in the future and on line 50 above.